e2e/mockserver/mockserver.ts

```diff
@@ -144,6 +144,11 @@ export class MockServer {
+                return;
+            }
```

Comment:
This looks like a leftover from testing and it makes your tests fail (because of double-encoded JSON).

Reply:
removed